includes/gateways/paypal/checkout-actions.php

```diff
@@ -424,3 +424,25 @@ function capture_order() {
 
 add_action( 'wp_ajax_nopriv_edd_capture_paypal_order', __NAMESPACE__ . '\capture_order' );
 add_action( 'wp_ajax_edd_capture_paypal_order', __NAMESPACE__ . '\capture_order' );
+
+/**
+ * Gets a fresh set of gateway options when a PayPal order is cancelled.
+ *
```

Comment:
Can you link to https://github.com/awesomemotive/easy-digital-downloads/issues/8883 so we can remember why we did this? :grin: